scripts/start.py

```diff
@@ -196,25 +196,35 @@ def main(args: Optional[Sequence[str]] = None) -> None:
             )
             initializer.populate_debug_data()
 
-        managed_web_browser = (
-            None if parsed_args.no_browser else
-            servers.create_managed_web_browser(PORT_NUMBER_FOR_GAE_SERVER))
-
-        if managed_web_browser is None:
+        if parsed_args.no_browser:
             common.print_each_string_after_two_new_lines([
                 'INFORMATION',
                 'Local development server is ready! You can access it by '
                 'navigating to http://localhost:%s/ in a web '
                 'browser.' % PORT_NUMBER_FOR_GAE_SERVER,
             ])
         else:
-            common.print_each_string_after_two_new_lines([
-                'INFORMATION',
-                'Local development server is ready! Opening a default web '
-                'browser window pointing to it: '
-                'http://localhost:%s/' % PORT_NUMBER_FOR_GAE_SERVER,
-            ])
-            stack.enter_context(managed_web_browser)
+            try:
+                stack.enter_context(servers.create_managed_web_browser(
+                    PORT_NUMBER_FOR_GAE_SERVER))
+                common.print_each_string_after_two_new_lines([
+                    'INFORMATION',
+                    'Local development server is ready! Opening a default web '
+                    'browser window pointing to it: '
+                    'http://localhost:%s/' % PORT_NUMBER_FOR_GAE_SERVER,
+                ])
+            except Exception as error:
```

Comment:
Are we catching all errors? Or do we want to capture a select few?

Reply:
Yes, we are catching all the errors that can occur with auto-launch of the web browser. The idea is allow the user to open the web browser manually if the server is running.